### ISSUE TITLE
Dropout between embeddings and RNN?

### ISSUE THREAD

**Issue by loopdigga98** [zalandoresearch/flair#516]
Hello. Thank you for this repo. I have some questions about ```flair/models/sequence_tagger_model.py``` module. 
Could you tell me what is the purpose of **dropout**, **word_dropout** and **locked_dropout**  in forward method (line 367) between creating batch of embeddings vectors and putting them in RNN layer. And how to deal with them on inference period?
Thank you.

**Comment by alanakbik** [comment]
Hi @loopdigga98 - these are different variants of [dropout](http://jmlr.org/papers/v15/srivastava14a.html) which is a method for preventing overfitting when training a neural network that works by randomly setting weights to zero during training. We apply it between the embedding, the RNN and the output layers. Dropout is deactivated during inference by default, so there is no need to deal with it. 

Does this answer the question?

**Comment by loopdigga98** [comment]
Yes, thank you.
